projects/plugins/jetpack/modules/subscriptions/subscribe-overlay/subscribe-overlay.css

```diff
@@ -13,19 +14,34 @@ body.jetpack-subscribe-overlay-open {
 	width: 100%;
 	height: 100%;
 	overflow: hidden;
-	background-color: transparent;
-	transition: background-color 0.4s, visibility 0.4s;
+	background-color: var(--jetpack-subscribe-overlay--background-color);
+	transition: opacity 0.4s, visibility 0.4s;
+
+	/* Those slyles should be come from the Site Editor */
+	.is-content-justification-center {
+		justify-content: center;
+	}
+
+	.is-content-justification-left {
+		justify-content: left;
+	}
+
+	.is-content-justification-right {
+		justify-content: right;
+	}
```

Comment:
Not a fan of this at all :(

Those styles should come from the Site Editor but they are missing. Same with the `max-width: 400px;` below.

Asked about it here p1718028247445989-slack-C03N25JPCE4 but no one had an idea what could be wrong...